backend/analytics_server/mhq/service/settings/configuration_settings.py

```diff
@@ -58,6 +59,13 @@ def _adapt_incident_types_setting_from_setting_data(
             ]
         )
 
+    def _adapt_default_sync_days_setting_setting_from_setting_data(
```

Comment:
"setting" word is repeated.
\_adapt_default_sync_days_<mark>setting_setting</mark>_from_setting_data

Reply:
Oh. good catch @Kamlesh72 
thanks mate